assets/packages/adminbasics/build/adminbasics.js

```diff
@@ -29781,6 +29781,31 @@
         if (e.target != this) return;
         $('#accordion').find('.panel-collapse.collapse').not('#' + $(this).attr('id')).collapse('hide');
       });
+    },
+
+    /**
+     * Validates that an end date is not lower than a start date
+     */
+    validateEndDateHigherThanStart: function validateEndDateHigherThanStart(startDatePicker, endDatePicker, errorCallback) {
```

Comment:
Hm, isn't this code duplication from a previous commit?

Reply:
Oh, you removed it. But you still have it in two places in this PR?

Reply:
Oh right : https://github.com/LimeSurvey/LimeSurvey/pull/2679/files#diff-ede93d7c4e63dba99f44a55b638edd64610bd2096268abc5375ba0bc21ffa149R119

2nd place